### DISCUSSION TITLE
Directory dump from failed minio s3 server ? How I can restore data ?

### DISCUSSION THREAD

**Discussion by master-747** [minio/minio#16033] [Q&A] [answered]
Hi 

If I had directory dump from failed minio s3 server.

How can I restore it back to working minio s3 ?

Data is like this

ls -ltr
total 0
drwxr-xr-x. 3 minio-user minio-user 77 Nov  9 14:08 'Screenshot 2022-11-07 at 7.33.57.png'
drwxr-xr-x. 3 minio-user minio-user 77 Nov  9 14:08 'Screenshot 2022-11-03 at 10.30.31.png'

ls *
'Screenshot 2022-11-03 at 10.30.31.png':
074dc455-f229-44e4-9a51-8520b40ce3a1  xl.meta

'Screenshot 2022-11-07 at 7.33.57.png':
22e6168c-208e-4992-b7c7-fac7b3b1c747  xl.meta

**Comment by klauspost** [comment, marked as answer]
There is no direct recovery. Your files are split across multiple drives.

Creating a similar setup and add the drives (or copy the data) it the most common way to recover.

See https://min.io/docs/minio/kubernetes/upstream/operations/data-recovery.html
